### DISCUSSION TITLE
Combine wildcard and other text on a line?

### DISCUSSION THREAD

**Discussion by Jonseed** [adieyal/sd-dynamic-prompts#278] [Q&A] [answered]
Can you combine a wildcard with other text on a line in a wildcard file, like this:

`__wildcard__ some other text`

Does that work?

**Comment by Jonseed** [comment]
I just answered my own question. It does appear to work.

**Comment by Jonseed** [comment]
@adieyal this seems to not be possible anymore in the latest version. Something like this doesn't work anymore it seems:

`{night|nighttime} dimly lit low key <lora:epiNoiseoffset_v2:1>`

Before this would pick one of the two options at the start of the line, and then add the other info on the line. Now it just picks night or nighttime, and ignores the rest of the line.

Would it be possible to get this functionality back to include the rest of the line in the wildcard file?

**Comment by Jonseed** [comment]
Actually, it looks like if it picks the second item, then it includes the rest of the line. If it picks the first item, then it ignores the rest of the line. Not sure what happens if there are more than two items, it probably only includes the rest of the line if it picks the last option.

**Reply by adieyal** [reply]
I'm not able to reproduce this. Could you confirm the version of the extension and dynamicprompts library you are using? You can find them under the help section. Also, do you see any errors in the terminal?

**Reply by Jonseed** [reply]
Weird. I just tried it again, and it seems to be working right now. I'll keep an eye on it.

**Comment by Jonseed** [comment, marked as answer]
@adieyal I figured out what was going on. I had two wildcard files that had the same filename, and it was picking from the other file. I wonder if there could be a warning in the console when this happens, when there are two or more files named the same, which might lead to undesired behavior, so that it would be easier to track down the problem.

**Reply by adieyal** [reply]
good idea - perhaps a verbose logging mode
